tests/integration/17.SecurityTag.Tests.ps1

```diff
@@ -69,11 +87,16 @@ Describe -Name "SecurityTagAssignment" -Tag "Get" -Fixture {
     } ## end context
 
     AfterAll {
-        #AfterAll block runs _once_ at completion of invocation regardless of number of tests/contexts/describes.
-        #Clean up anything you create in here.  Be forceful - you want to leave the test env as you found it as much as is possible.
-        #We kill the connection to NSX Manager here.
+        # AfterAll block runs _once_ at completion of invocation regardless of number of tests/contexts/describes.
+        # Clean up anything you create in here.  Be forceful - you want to leave the test env as you found it as much as is possible.
+        # We kill the connection to NSX Manager here.
 
         if ($pause) {Read-Host "pausing" }
```

Comment:
remove the pause ?

Reply:
Greets-  seems fine to me -- I only kept it here as it was in the "template" test code upon which I based this overall test file.  Let me know, and I'll remove it.